src/shared/histogram.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi'
+import Metrics from '@mojaloop/central-services-metrics'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type THandlerFunc = (request: Request, h: ResponseToolkit) => Promise<ResponseObject>
+
+/**
+ * @function wrapWithHistogram
+ * @description Wraps a handler function with a histogram of the given name
+ * @param {THandlerFunc} handler The handler function to be wrapped
+ * @param {[string, string, Array<string>]} histogramParams The params of the histogram
+ */
+function wrapWithHistogram (handler: THandlerFunc, histogramParams: [string, string, string[]]): THandlerFunc {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
where do we have `any` here? drop the linting exception

Reply:
Sure, I can remove it. Not sure why it's there either.
I was wondering if I need the histogram for auth-service?
I removed the histogram in my latest commit but I can add it back